### PR TITLE
[DOC]: Create Alchemy's Road to Web3 chapter

### PR DESCRIPTION
**Pull request by mkubdev**
### Describe the changes you've made

Added Alchemy's RoadToWeb3 chapter.

## Type of change

What sort of change have you made:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update.
- [ ] This change requires a documentation update
 
## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please add `.md` at the end of the filename. It is missing the Markdown format. 🙂


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Comment by mkubdev** [comment]
> Please add `.md` at the end of the filename. It is missing the Markdown format. 🙂
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix  |  **Tags:** Not maintainable

Lol! Oopsy... i'll save that one.

File: docs/💖 Free Training/Alchemy's Road to Web3.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+[How to Build a "Buy Me a Coffee" DeFi dApp](https://docs.alchemy.com/docs/2-how-to-build-buy-me-a-coffee-defi-dapp)
+[How to Make NFTs with On-chain Metadata with Hardhat and Javascript](https://docs.alchemy.com/docs/3-how-to-make-nfts-with-on-chain-metadata-hardhat-and-javascript)
+[How to Create an NFT Gallery](https://docs.alchemy.com/docs/5-connect-apis-to-your-smart-contracts-using-chainlink)
+[How to Create a Dynamic NFT](https://docs.alchemy.com/docs/5-connect-apis-to-your-smart-contracts-using-chainlink)
+[How to Build a Staking dApp](https://docs.alchemy.com/docs/6-how-to-build-a-staking-dapp)
+[How to Build an NFT Marketplace from Scratch](https://docs.alchemy.com/docs/7-how-to-build-an-nft-marketplace-from-scratch)
+[How to Build a Betting Game on Optimism](https://docs.alchemy.com/docs/8-how-to-build-a-betting-game-on-optimism)
+[How to Build a Token Swap Dapp with 0x API](https://docs.alchemy.com/docs/9-how-to-build-a-token-swap-dapp-with-0x-api)
+[How to Create a Decentralized Twitter with Lens Protocol](https://docs.alchemy.com/docs/how-to-create-a-decentralized-twitter-with-lens-protocol)
```
Comment (Panquesito7) [inline docs/💖 Free Training/Alchemy's Road to Web3.md:15]: What if you make this a list by using `-`?
```suggestion
- [How to Develop an NFT Smart Contract (ERC721) with Alchemy](https://docs.alchemy.com/docs/1-how-to-develop-an-nft-smart-contract-erc721-with-alchemy)
- [How to Build a "Buy Me a Coffee" DeFi dApp](https://docs.alchemy.com/docs/2-how-to-build-buy-me-a-coffee-defi-dapp)
- [How to Make NFTs with On-chain Metadata with Hardhat and Javascript](https://docs.alchemy.com/docs/3-how-to-make-nfts-with-on-chain-metadata-hardhat-and-javascript)
- [How to Create an NFT Gallery](https://docs.alchemy.com/docs/5-connect-apis-to-your-smart-contracts-using-chainlink)
- [How to Create a Dynamic NFT](https://docs.alchemy.com/docs/5-connect-apis-to-your-smart-contracts-using-chainlink)
- [How to Build a Staking dApp](https://docs.alchemy.com/docs/6-how-to-build-a-staking-dapp)
- [How to Build an NFT Marketplace from Scratch](https://docs.alchemy.com/docs/7-how-to-build-an-nft-marketplace-from-scratch)
- [How to Build a Betting Game on Optimism](https://docs.alchemy.com/docs/8-how-to-build-a-betting-game-on-optimism)
- [How to Build a Token Swap Dapp with 0x API](https://docs.alchemy.com/docs/9-how-to-build-a-token-swap-dapp-with-0x-api)
- [How to Create a Decentralized Twitter with Lens Protocol](https://docs.alchemy.com/docs/how-to-create-a-decentralized-twitter-with-lens-protocol)
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

File: docs/💖 Free Training/Alchemy's Road to Web3.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Learn about [Alchemy's Road to Web3 series](https://docs.alchemy.com/docs/welcome-to-the-road-to-web3), a 10-week, self-paced web3 developer program to help new blockchain devs go from beginner to advanced.
+
+## Road to Web3 Learning Challenges
```
Comment (Panquesito7) [inline docs/💖 Free Training/Alchemy's Road to Web3.md:3]: There's a missing space here (Markdown Linter).
```suggestion
## Road to Web3 Learning Challenges :rocket:

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

File: docs/💖 Free Training/Alchemy's Road to Web3.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Learn about [Alchemy's Road to Web3 series](https://docs.alchemy.com/docs/welcome-to-the-road-to-web3), a 10-week, self-paced web3 developer program to help new blockchain devs go from beginner to advanced.
```
Comment (Panquesito7) [inline docs/💖 Free Training/Alchemy's Road to Web3.md:1]:
```suggestion
# Alchemy's road to Web3 :globe:

Learn about [Alchemy's Road to Web3 series](https://docs.alchemy.com/docs/welcome-to-the-road-to-web3), a 10-week, self-paced web3 developer program to help new blockchain devs go from beginner to advanced.
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Inelegant

**Comment by Panquesito7** [comment]
Any ideas about why the CI is failing and how can we fix that? 🤔

File: docs/💖 Free Training/Alchemys Road to Web3.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Alchemy's road to Web3 :globe:
```
Comment (Panquesito7) [inline docs/💖 Free Training/Alchemys Road to Web3.md:1]: Whoops, wrong emoji name. 😅
```suggestion
# Alchemy's road to Web3 :globe_with_meridians:
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Panquesito7 — DISMISSED** [review]

**Review by Panquesito7 — APPROVED** [review]
Awesome! Good to go. 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Pull request merged**